### PR TITLE
test CI

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: .teamcity/build_xly.sh [hunk 1]
```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/build_xly.sh]: 2018 -> 2022

File: .teamcity/Dockerfile [hunk 2]
```diff
@@ -15,10 +15,10 @@
 
 # A dev image based on paddle production image
 
-FROM parl/parl-test:2.0-cuda10.1-v1
+FROM parl/parl-test:2.0-cuda10.1-v2_xly
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/Dockerfile:18]: What does xly mean? ...

File: .teamcity/build_xly.sh [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+function init() {
+    RED='\033[0;31m'
+    BLUE='\033[0;34m'
+    BOLD='\033[1m'
+    NONE='\033[0m'
+
+    REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}")/../" && pwd )"
+
+    export PATH="/root/miniconda3/bin:$PATH"
+    export LD_LIBRARY_PATH="/usr/local/TensorRT-6.0.1.5/lib:$LD_LIBRARY_PATH"
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/build_xly.sh]: Is there a GPU in the testing environment?

File: .teamcity/Dockerfile (hunk 2 above)
Comment (zenghsh3) [reply .teamcity/Dockerfile:18]: 效率云

File: .teamcity/build_xly.sh (hunk 3 above)
Comment (zenghsh3) [reply .teamcity/build_xly.sh]: yes

**Pull request closed without merging**